### ISSUE TITLE
about accuracy of GAT

### ISSUE THREAD

**Issue by FOURIERWANG** [dmlc/dgl#1163]
The accuracy of the citeseer dataset is 2% worse than the paper published in the GAT method, why  so much?

**Comment by yzh119** [comment]
Hi,
Our result on Citeseer has a large variance, the highest accuracy we can get is 73.6 while the lowest accuracy we get is 69.8, depend on the random seed we select.
So I suppose only with a better hyper-parameter setting can we reproduce the average accruacy reported.

**Comment by VoVAllen** [comment]
Also GAT needs early stopping, and you may need to turn the hyperparameter for early stopping

**Comment by yzh119** [comment]
What I reported is the result after early stopping. It's still unstable..

Best regards,
Zihao
